code/game/objects/items/handcuffs.dm

```diff
@@ -61,28 +61,40 @@
 								"<span class='userdanger'>[user] is trying to put [src.name] on you!</span>")
 
 			playsound(loc, cuffsound, 30, TRUE, -2)
-			if(do_mob(user, C, 30) && (C.get_num_arms(FALSE) >= 2 || C.get_arm_ignore()))
+			if(do_mob(user, C, 30))
 				if(iscyborg(user))
 					apply_cuffs(C, user, TRUE)
 				else
 					apply_cuffs(C, user)
-				C.visible_message("<span class='notice'>[user] handcuffs [C].</span>", \
-									"<span class='userdanger'>[user] handcuffs you.</span>")
-				SSblackbox.record_feedback("tally", "handcuffs", 1, type)
-
-				log_combat(user, C, "handcuffed")
 			else
 				to_chat(user, "<span class='warning'>You fail to handcuff [C]!</span>")
 		else
-			to_chat(user, "<span class='warning'>[C] doesn't have two hands...</span>")
+			to_chat(user, "<span class='warning'>[C == user ? "You don't" : "[C] doesn't"] have two hands...</span>")
+	else
+		to_chat(user, "<span class='warning'>[C == user ? "You're" : "[C] is"] already handcuffed!</span>")
+
+/obj/item/restraints/handcuffs/proc/apply_cuffs(mob/living/carbon/target, mob/user, dispense = FALSE, intentional = FALSE)

```

Comment:
This will need documentation.

Reply:
is the documentation that's here insufficient?

Reply:
The proc has no documentation.
Here's the guide:
https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md